File: src/main/java/com/tomath/resource/ClientesResource.java
package com.tomath.resource;

import com.tomath.model.Cliente;
import com.tomath.repository.Clientes;
import org.springframework.beans.factory.annotation.Autowired;
import org.springframework.web.bind.annotation.GetMapping;
import org.springframework.web.bind.annotation.RequestMapping;
import org.springframework.web.bind.annotation.RestController;

import java.util.List;


@RestController
@RequestMapping("/clientes")
public class ClientesResource {
	
	@Autowired
	private Clientes clientes;
	
	@GetMapping
	public List<Cliente> listar(){
		return this.clientes.findAll();
	}
}

File: src/main/kotlin/com/tomath/cliente/ClientesResource.kt
package com.tomath.cliente

import org.springframework.web.bind.annotation.GetMapping
import org.springframework.web.bind.annotation.RequestMapping
import org.springframework.web.bind.annotation.RestController

@RestController
@RequestMapping("/clientes")
class ClientesResource(private val clientes: ClienteRepositorio) {
    @GetMapping
    fun listar(): List<Cliente?> {
        return clientes.findAll()
    }
}